notebooks/learning-to-rank/01-learning-to-rank.ipynb

```diff
@@ -0,0 +1,1556 @@
+            "text/plain": [
+              "<Figure size 640x480 with 1 Axes>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "xgb.plot_importance(ranker, importance_type='weight')\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Importing the model to Elasticsearch\n",
+        "\n",
+        "Once the model is trained you will be able to use Eland to send it to Elasticsearch.\n",
+        "\n",
+        "Please note that the `MLModel.import_ltr_model` method contains the LTRModelConfig object, so you do not need to send it separately to configure feature extraction.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 14,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "zAMwvqYlq9py",
+        "outputId": "c0f60ce3-fb07-47a5-9e37-fccbd1f30bcc"
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "<eland.ml.ml_model.MLModel at 0x28222a940>"
+            ]
+          },
+          "execution_count": 14,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "from eland.ml import MLModel\n",
+        "\n",
+        "MLModel.import_ltr_model(\n",
+        "  es_client=es_client,\n",
+        "  model=ranker,\n",
+        "  model_id='ltr-model-xgboost',\n",
```

Comment:
this name is used again further down, so we might want a variable for it

Reply:
Added a variable.